### PR TITLE
implement state=absent

### PR DESCRIPTION
**Pull request by fmenabe**

### CODE REVIEW

**Comment by zauberpony** [comment]
Hey, great job, thank's for your work, looks good, I just added one small remark/question.

**Comment by fmenabe** [comment]
Sorry but what/where are your remark/question?

**Comment by zauberpony** [comment]
Sorry, forgot to finish (and publish) the review :D

File: library/mysql_query.py [hunk 1]
```diff
@@ -282,8 +295,8 @@ def main():
     if not mysqldb_found:
         module.fail_json(msg="the python mysqldb module is required")
 
-    if module.params["state"] == "absent":
-        module.fail_json(msg="state=absent is not yet implemented")
+    if module.params["state"] == "absent" and module.params["values"]:
+        module.fail_json(msg="state=absent does not need values")
```
[the remark this answers is not in the document]
Comment (fmenabe) [reply library/mysql_query.py]: I not sure there really is a practice for managing unused arguments and for what I've seen it depends on the module developper (but I may be wrong). Anyway, I have no opinion on this and will delete the check.

**Comment by fmenabe** [comment]
The PR has been updated.

And thanks for your reactivity!

**Comment by zauberpony** [comment]
Sure, thanks for work, again.

This fixes #7

**Pull request merged**